lib/runners/processor/pmd_cpd.rb

```diff
@@ -0,0 +1,199 @@
+module Runners
+  class Processor::PmdCpd < Processor
+    include Java
+
+    Schema = StrongJSON.new do
+      let :runner_config, Schema::BaseConfig.base.update_fields { |fields|
+        fields.merge!({
+                    minimum_tokens: numeric?,
+                    files: string?,
+                    filelist: string?,
```

Comment:
[suggestion] How about specifying target files or directories more easily?

```suggestion
                    files: enum(string, array(string)),
```

E.g. Support only `files` (required):

```yaml
linter:
  pmd_cpd:
    files: src #=> --files src
    # or
    files: [src, lib] #=> --files src --files lib
    # or
    files: [path/to/Some.java, src] #=> --files path/to/Some.java --files src
```

Ruby code:

```ruby
Array(config_linter[:files] || DEFAULT_FILES).map { |v| ["--files", v] }
```

See https://pmd.github.io/latest/pmd_userdocs_cpd.html#cli-usage

Reply:
Good call!
Fixed. commit 8cc0eb507a0a1c0bac6e954c2f995c8cb7ed8349